Source/TickerCounter.swift

```diff
@@ -247,6 +257,18 @@ public final class TickerCounter: UIView {
         }
     }
     
+    private func generateMonospacedStringWidth(characterSize: CGSize) -> CGFloat {
```

Comment:
this doesn't necessarily belong as `TickerCounter` behavior - consider refactoring somewhere else

Reply:
Moved!